packages/react-components/src/components/FileDownloadCards.tsx

```diff
@@ -193,7 +193,9 @@ export const _FileDownloadCards = (props: _FileDownloadCards): JSX.Element => {
   if (
     !fileMetadata ||
     fileMetadata.length === 0 ||
-    /* @conditional-compile-remove(teams-inline-images) */ fileMetadata[0].attachmentType !== 'fileSharing'
+    /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */ !fileMetadata.some(
```

Comment:
If we are making this change it is safe to remove the above comment 
```
// Its safe to assume that if the first item in the fileMetadata is not a fileSharing type we don't want to display the FileDownloadCard.
  // Since you can't have both fileSharing and teamsInlineImage in the same message.
```

Reply:
Or at least update the comment so we understand why this check is in there.